tests/Backend/Workspaces/WorkspacesSnowflakeTest.php

```diff
@@ -1065,4 +1069,97 @@ public function testTableLoadAsView(): void
             $this->assertStringContainsString('does not exist or not authorized', $e->getMessage());
         }
     }
+
+    public function testLoadFromBranch(): void
```

Comment:
strcil jsem to sem, bo vytvarim branchovany bucket, coz je mozny, jen ve snflk a jinde to zatim neni blokovane